defaults/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 
 ## General
-solr_version: 8.2.0
+solr_version: 8.3.0
```

Comment:
This change is not reflected in changelog :p

Reply:
done